src/Parallel/CharmRegistration.hpp

```diff
@@ -356,6 +362,15 @@ struct get_value_type<
   using type = std::pair<Key, Mapped>;
 };
 
+template <size_t Size, class Key, class Mapped, class Hash, class KeyEqual>
+struct get_value_type<FixedHashMap<Size, Key, Mapped, Hash, KeyEqual>> {
+  // When sending data it is typical to use `std:make_pair(a, b)` which results
+  // in a non-const Key type, which is different from what
+  // `FixedHashMap::value_type` is. This difference leads to issues with
```

Comment:
it would be helpful to include in the comment the type that `FixedHashMap::value_type` actually is. So, possibly include in the comment `(i.e. std::pair<const Key, Mapped>)`.